src/components/slider/_slider.scss

```diff
@@ -0,0 +1,77 @@
+//-----------------------------
+// Slider
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/helper-mixins';
+@import '../../globals/scss/layer';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/layout';
+@import '../../globals/scss/import-once';
+@import '../form/form';
+
+@include exports('slider') {
+  .bx--slider-container {
+    width: 30rem;
+    display: flex;
+    align-items: center;
+  }
+
+  .bx--slider {
+    position: relative;
+    width: 100%;
+  }
+
+  .bx--slider--disabled {
+    opacity: .5;
+    & > * {
+      cursor: not-allowed !important; // fix
+    }
+  }
+
+  .bx--slider__track {
+    position: absolute;
+    width: 100%;
+    height: rem(4px);
+    background: $ui-05;
+    cursor: pointer;
+  }
+
+  .bx--slider__filled-track {
+    position: absolute;
+    width: 100%;
+    height: rem(4px);
+    background: $brand-01;
+    transform-origin: left;
+    pointer-events: none;
+  }
+
+  .bx--slider__thumb {
+    position: absolute;
+    height: rem(24px);
+    width: rem(24px);
+    background: $brand-01;
+    border-radius: 50%;
+    top: 0;
+    transform: translate(-50%, -50%);
+    transition: transform 100ms $bx--standard-easing;
+    cursor: pointer;
+    outline: none;
+
+    &:hover, &:focus {
+      transform: translate(-50%, -50%) scale(1.25);
```

Comment:
slider__thumb is also missing a dedicated focus state. 
You can use this code to handle focus and high-contrast mode styles:

```scss
.bx--slider__track {
  outline: 1px solid transparent;
}

.bx--slider__thumb {
  &:focus {
    @include focus-outline('blurred');
    outline: 2px solid transparent;
  }
}
```

Double-check with visual to make a final decision on this.